soql-analyzer/src/main/scala/com/socrata/soql/AnalysisDeserializer.scala

```diff
@@ -266,61 +227,43 @@ class AnalysisDeserializer[C, T](columnDeserializer: String => C, typeDeserializ
         dictionary.strings(in.readUInt32())
       }
 
-    def readAnaAndFrom(): (SoQLAnalysis[C, T], Option[String]) = {
-      lazy val fromOptTrigger = if (isV4) maybeRead(in.readString) else None
+    def readAnaAndFrom(): SoQLAnalysis[C, T] = {
 
       val ig = readIsGrouped()
       val d = readDistinct()
       val s = readSelection()
-      val j = {
-        fromOptTrigger; // trigger eval, read "from" string if v4
-        readJoins()
-      }
+      val j = readJoins()
       val w = readWhere()
-      val gb = bcSeq { readGroupBy() }
+      val gb = readGroupBy()
       val h = readHaving()
-      val ob = bcSeq { readOrderBy() }
+      val ob = readOrderBy()
       val l = readLimit()
       val o = readOffset()
       val search = readSearch()
 
-      val ana = SoQLAnalysis(ig, d, s, j, w, gb, h, ob, l, o, search)
-
-      (ana, fromOptTrigger)
+      SoQLAnalysis(ig, d, s, j, w, gb, h, ob, l, o, search)
     }
 
     def readAnalysis(): SoQLAnalysis[C, T] = {
-      readAnaAndFrom()._1
+      readAnaAndFrom()
```

Comment:
looks like the contents of `readAnaAndFrom` could be collapsed into this function.